captions.csv

```diff
@@ -8,10 +8,13 @@ Rolling with the homies.,Happy,"friends, people, fun",Positive
 The most important thing is to enjoy your life – to be happy – it’s all that matters.,Happy,"travel, work, scenary",Positive
 50% Savage. 50% Sweetness.,Neutral,"Selfie, person, badass, cute",Neutral
 Watch more sunsets than Netflix.,Happy,"Calm, sunset, view, mountains",Positive
-"Love can be unselfish, in the sense of being benevolent and generous, without being selfless.",,,Positive
-"When I feel a little down, I put on my favourite high heels and dance",,,Positive
-Pretty please with a cherrry on top,,,Positive
```

Comment:
Don't remove the existing captions.

Reply:
ok i will add more captions without removing others